docs/asciidoc/more_background_on_compact_identifiers_content.adoc

```diff
@@ -0,0 +1,116 @@
+
+    GET https://registry.api.identifiers.org/restApi/resources/search/findAllByNamespaceId?id=75
+
+This returns enough information to, ultimately, identify one or more resolvers and each have a URL pattern that, for DRS-supporting systems, provides a URL template for making a successful DRS GET request.  For example, the DOI urlPattern is:
+
+    urlPattern: "https://doi.org/{$id}"
+
+And the `{$id}` here refers to the accession from the compact identifier (in this example the accession is `10.5072/FK2805660V`).  If applicable, a provide code can be supplied in the above requests to specify a particular mirror if there are multiple resolvers for this namespace.  In the case of DOIs, you only get a single resolver.
+
+Given this information you now know you can make a GET on the URL:
+
+    GET https://doi.org/10.5072/FK2805660V
+
+_The URL above is valid for a DOI object but it is not actually a DRS server!  Instead, it redirects to a DRS server through a series of HTTPS redirects.  This is likely to be common when working with existing compact identifiers like DOIs or ARKs.  Regardless, the redirect should eventually lead to a DRS URL that percent-encodes the accession as a DRS ID in a DRS object API call.  For a **hypothetical** example, here's what a redirect to a DRS API URL might ultimately look.  A client doesn't have to do anything other than follow the HTTPS redirects.  The link between the DOI resolver on doi.org and the DRS server URL below is the result of the DRS server registering their data objects with a DOI issuer._
+
+    GET https://drs.example.org/ga4gh/drs/v1/objects/10.5072%2FFK2805660V
+
+IDs in DRS hostname-based URIs/URLs are always percent-encoded to eliminate ambiguity even though the DRS compact identifier-based URIs and the identifier.orgs API do not percent-encode accessions.  This was done in order to 1) follow the CURIE conventions of identifiers.org/n2t.net for compact identifier-based DRS URIs and 2) to aid in readability for users who understand they are working with compact identifiers. **The general rule of thumb, when using a compact identifier accession as a DRS ID in a DRS API call, make sure to percent-encode it.  An easy way for a DRS client to handle this is to get the initial DRS object JSON response from whatever redirects the compact identifier resolves to, then look for the `self_uri` in the JSON, which will give you the correctly percent-encoded DRS ID for subsequent DRS API calls such as the `access` method.**
+
+
+== Example DRS Client Compact Identifier-Based URI Resolution Process - Registering a new Compact Identifier for Your DRS Server
```

Comment:
Let's move this section out of the spec and into the "more background" doc -- it will remove a lot of clutter, is only relevant to a minority of our users, and is the most likely to get out of date as other systems change.